blog/20230722-simplex-chat-v5-2-message-delivery-receipts.md

```diff
@@ -13,26 +13,135 @@ permalink: "/blog/20230722-simplex-chat-v5-2-message-delivery-receipts.html"
 
 **Published:** July 22, 2023
 
-TODO this is a draft of the release announcement
-
 What's new in v5.2:
+- [message delivery receipts](#message-delivery-receipts) – with opt-out per contact!
+- [filter favorite and unread chats](#filter-favorite-and-unread-chats).
+- a bit [more usable groups](#more-usable-groups):
+  - view full original replied to message in info.
+  - share your address with your contacts and group members via your chat profile.
+  - search for new and existing members.
+- [stability improvements](#stability-improvements):
+  - keep your connections working after restoring from backup.
```

Comment:
+ ability to abort connection switch, fix behavior that allowed switching multiple time